torchao/quantization/prototype/qat/linear.py

```diff
@@ -26,7 +24,11 @@
 from torchao.quantization.utils import get_group_qparams_symmetric
 from .api import FakeQuantizeConfig
 from .fake_quantizer import FakeQuantizer
-from .utils import (
+from torchao.quantization.utils import (
+    get_groupwise_affine_qparams,
+    groupwise_affine_quantize_tensor,
+)
+from torchao.quantization.prototype.qat.utils import (
```

Comment:
sorry this is recently updated to `torchao.quantization.qat.utils`

Reply:
Oh actually you'll face some conflicts with this file. Can you make the changes in `torchao/quantization/qat/linear.py` instead? Sorry for the merge conflicts